client/utils/currency/index.js

```diff
@@ -219,7 +220,7 @@ function formatExchangeRate( from, to ) {
 			to.currency.toUpperCase()
 		);
 	}
-	const exchangeCurrency = new Currency( {
+	const exchangeCurrency = Currency( {
```

Comment:
It used to be a class (with a constructor) in older package versions, but it's just a function since then.